components/ProductImage.js

```diff
@@ -0,0 +1,13 @@
+app.component('product-image', {
+    props: {
+        image: {
+            type: String
+        }
+    },
+    template:
+        /*html*/
+        `<div class="product-image">
```

Comment:
correct alignment pls

Reply:
You are making my life Reallyyyy diffiicult

Reply:
you like it